vmaas_sync/metrics.go

```diff
@@ -96,26 +99,58 @@ func update() {
 }
 
 func updateSystemMetrics() {
-	optOuted, notOptOuted, err := getSystemCounts()
+	counts, err := getSystemCounts(time.Now())
 	if err != nil {
 		utils.Log("err", err.Error()).Error("unable to update system metrics")
+		return
+	}
+
+	for key, count := range counts {
+		labels := strings.Split(key, ":")
```

Comment:
Why not use a simple struct here instead of concatenating & splitting strings ?

Reply:
yes, good idea, thanks, done